Kudu.Core/Deployment/StructuredTextLogger.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Kudu.Core.Tracing;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Kudu.Core.Deployment
+{
+    public class StructuredTextLogger : IDetailedLogger
+    {
+        private readonly static object DocumentLock = new object();
```

Comment:
so entire application life cycle we only have one instance of "StructuredTextLogger" ?